design/common.blocks/button/_theme/button_theme_islands.styl

```diff
@@ -142,21 +142,35 @@ ctx = '.button_theme_islands'
 
     &.button_view_action
     {
-        background: rgba(191, 153, 0, 0.8);
-
         &:before
         {
+            top: 0;
+            right: 0;
+            bottom: 0;
+            left: 0;
+
+            border-radius: 3px;
             background: #ffdb4d;
         }
 
+        &.button_hovered:before
+        {
+            background: #ffd633;
+        }
+
         &.button_pressed:before
         {
             background: #fc0;
         }
 
-        &.button_hovered
+        &.button_focused
         {
-            background: rgba(159, 127, 0, 0.8);
+            box-shadow: none;
+
+            &:before
+            {
+                animation: button-action-focus 0.5s infinite linear alternate;
```

Comment:
It worth to add `background-color` for browsers that don't support animations (IE9). `animation` has top priority, thus it'll not affect supporting browsers.